.vscode/launch.json

```diff
@@ -125,6 +125,28 @@
         "NODE_ENV": "development"
       },
       "sourceMaps": true
+    },
+    {
+      "type": "node",
```

Comment:
Did you mean to check this in? I'm guessing so but I just wanted to make sure.

Reply:
no, I think this was just from testing with jest. Will revert this